apps/src/applab/commands.js

```diff
@@ -199,12 +199,12 @@ applabCommands.button = function(opts) {
   var textNode = document.createTextNode(opts.text);
   newButton.id = opts.elementId;
   newButton.style.position = 'relative';
-  newButton.style.fontSize = defaultFontSizeStyle;
-  newButton.style.fontFamily = fontFamilyStyles[0];
```

Comment:
Hm - was going to ask why we don't apply the special theme-specific sizing logic here, but might see it already.  It looks like these two button-creation paths (via design mode, via code) have never quite been unified and it might break existing apps to unify them now.  Default button size is already different for the two paths.

Reply:
Correct, and I didn't want to change that. The 80x30 default size only applied to design mode created buttons.